Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionWatchGuardFirewareOS.yaml

```diff
@@ -0,0 +1,84 @@
+Parser:
+  Title: Network Session ASIM parser for WatchGuard Fireware OS
+  Version: '0.1.0'
+  LastUpdated: Aug 10th, 2022
+Product:
+  Name: WatchGuard Fireware OS
+Normalization:
+  Schema: NetworkSession
+  Version: '0.2.4'
+References:
+- Title: ASIM Network Session Schema
+  Link: https://aka.ms/ASimNetworkSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing WatchGuard Fireware OS logs to the ASIM Network Session normalized schema.
+ParserName: ASimNetworkSessionWatchGuardFirewareOS
+EquivalentBuiltInParser: _ASim_NetworkSession_WatchGuardFirewareOS
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventLookup=datatable(DvcAction:string,EventResult:string,EventSeverity:string)
+  [
+      "Allow","Success","Informational"
+      , "Deny","Failure","Low"
+  ];
+  let Parser=(disabled:bool=false){
+  let AllLogs = Syslog
+  | where SyslogMessage has_any('msg_id="3000-0148"' 
+      , 'msg_id="3000-0149"' 
+      , 'msg_id="3000-0150"'
+      , 'msg_id="3000-0151"'
+      , 'msg_id="3000-0173"'
+  )
+  | project TimeGenerated, SyslogMessage, HostName;

```

Comment:
I think that in Syslog it can be HostName or Computer. Never really figured out when each is used.

Reply:
As far as I can see it can indeed be both.